browser-app.cpp

```diff
@@ -83,6 +83,19 @@ void BrowserApp::OnBeforeCommandLineProcessing(
 
 	command_line->AppendSwitch("enable-system-flash");
 
+	if (command_line->HasSwitch("disable-features")) {
+		// Don't override existing, as this can break OSR
+		std::string disableFeatures = 
+			command_line->GetSwitchValue("disable-features");
+				disableFeatures = 
+					disableFeatures += ",HardwareMediaKeyHandling";
```

Comment:
double assignment to disableFeatures
`disableFeatures += ",HardwareMediaKeyHandling";`
or
`disableFeatures = disableFeatures + ",HardwareMediaKeyHandling";`

Reply:
Nice catch!